core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpAdapter.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.help
+
+import android.animation.ObjectAnimator
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import butterknife.BindView
+import butterknife.ButterKnife
```

Comment:
We don't use `butterknife` in kotlin. Use kotlinSynthetics as you have done so for others

Reply:
@s-ayush2903 sorry for that.I have to use Butterknife because at compiling time  lint shows the error UnusedId.I don't know,How to prevent from that?

Reply:
@gouri-panda post your exact error here, also look to `BaseViewHolder` and its subclasses for an idea of how we bind data to views. FileViewHolder might be closest to this situation

Reply:
Everything works perfectly. but the lint shows this error.
```
 The resource R.id.item_help_description appears to be unused [UnusedIds]
```

Reply:
post the code, commit with a `--no-verify` if you need to